app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -133,7 +133,7 @@ def manifest
     private
 
       def user_collections
-        collections_service.search_results(:deposit) if presenter.editor?
+        collections_service.search_results(:deposit)
```

Comment:
The restriction to `if presenter.editor?` prevents viewers of the work to have access to their collections.